harness/determined/cli/workspace.py

```diff
@@ -147,6 +178,21 @@ def list_pools(args: argparse.Namespace) -> None:
     )
 
 
+def list_workspace_members(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    resp = bindings.get_GetGroupsAndUsersAssignedToWorkspace(sess, workspaceId=w.id)
+
+    if args.json:
+        render.print_json(resp.to_json())
+    elif args.yaml:
```

Comment:
Nice. I like that there is a yaml option even though not all other CLI items have this.